common/models/user.json

```diff
@@ -206,6 +206,10 @@
       "type": "string",
       "description": "An IETF language tag",
       "default": "en"
+    },
+    "coreTeam": {
```

Comment:
Let's change the structure to be more extensible.

```json
  badges: {
    coreTeam: [
      "2016"
    ]
  } 
```

This way we can add more badges in the future